### ISSUE TITLE
Elpy's yasnippets can now be unbundled

### ISSUE THREAD

**Issue by sten0** [jorgenschaefer/elpy#2040]
Elpy's yasnippets have been merged upstream and can now be unbundled.  This will lighten the maintenance burden.

https://github.com/AndreaCrotti/yasnippet-snippets/commit/9f67370d9dbff73b0a4bf7c859455659b193f991

**Comment by yitang** [comment]
good idea, but be mindful there are differences, e.g. cls in elpy is more comprehensive, but merged cls in yasnippet-snippets is barebone.
